CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### 6.0.0
```

Comment:
**thought:** Sorry I haven't documented this anywhere, but previously we've had in-progress (unreleased) changes as `### vNext`. I think it's good to indicate that this is a major version change, so maybe we can do something like: 

```md
### 6.x (work in progress)
```

or

```md
### vNext (6.x)
```

WDYT?

Reply:
make sense. PR is updated